include/proxy-wasm/wasm_vm.h

```diff
@@ -77,6 +77,7 @@ template <size_t N> using WasmCallbackWord = WasmFuncType<N, Word, void *, Word>
 // Z = void, j = uint32_t, l = int64_t, m = uint64_t
 using WasmCallback_WWl = Word (*)(void *, Word, int64_t);
 using WasmCallback_WWlWW = Word (*)(void *, Word, int64_t, Word, Word);
+using WasmCallback_WWlW = Word (*)(void *, Word, int64_t, Word);
```

Comment:
Nit: `WasmCallback_WWlW` should be between `WasmCallback_WWl` and `WasmCallback_WWlWW`.